config/module.config.php

```diff
@@ -1,13 +1,13 @@
 <?php
-return array(
-    'service_manager' => array(
-        'factories' => array(
-            'ZF\HttpCache\HttpCacheListener' => 'ZF\HttpCache\HttpCacheListenerFactory',
-        ),
-    ),
+return [
+    'service_manager' => [
+        'factories' => [
+            \ZF\HttpCache\HttpCacheListener::class => \ZF\HttpCache\HttpCacheListenerFactory::class,
```

Comment:
To make things more readable, I often import such classes at the top of the file.